vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -278,6 +278,11 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 			Computed:    true,
 			Description: "The machine object ID from VMWare",
 		},
+                "power_state": {
+                        Type:        schema.TypeString,
+                        Computed:    true,
+                        Description: "Power state for the virtual machine.",
```

Comment:
```go
Description: "Power state of the virtual machine.",
```

Reply:
Resolvd in https://github.com/hashicorp/terraform-provider-vsphere/pull/1407/commits/4925157671e97d52889181d977cad8123b92473d